terraform/groups/ecs-service/locals.tf

```diff
@@ -0,0 +1,74 @@
+# Define all hardcoded local variable and local variables looked up from data resources
+locals {
+  stack_name                 = "public-data" # this must match the stack name (cluster) the service deploys into
+  name_prefix                = "${local.stack_name}-${var.environment}"
+  global_prefix              = "global-${var.environment}"
+  service_name               = "company-metrics-api"
+  container_port             = "8080"
+  eric_port                  = "10000"
+  docker_repo                = "company-metrics-api"
+  kms_alias                  = "alias/${var.aws_profile}/environment-services-kms"
+  lb_listener_rule_priority  = 144
+  lb_listener_paths          = ["/company/(.*)/metrics/(.*)", "/company/(.*)/metrics", "/company-metrics-api/healthcheck"]
```

Comment:
I think we probably want these written like /company/*/metrics/* etc

Reply:
resolved